cypress/e2e/OrangeHRM/TC-05_VerifyUpdateEmployeeOnOrangeHRM/TC-05_VerifyUpdateEmployeeOnOrangeHRM.ts

```diff
@@ -36,29 +36,31 @@ Given("the user add a new employee", () => {
 When("the user updatde the added employee", () => {
     updateEmployeeAPI.UpdateEmployee(
         updateEmployee = {
```

Comment:
It's better for this to declare it outside the API , I mean after line 36 since you need the emp number  you can declare variable here and pass it in the API